extensions/src/main/java/org/pentaho/platform/plugin/services/security/userrole/ldap/DefaultLdapAuthenticationProvider.java

```diff
@@ -78,7 +80,11 @@ protected Collection<? extends GrantedAuthority> loadUserAuthorities( DirContext
 
   @Override
   public Authentication authenticate( Authentication authentication ) throws AuthenticationException {
-    final Authentication authenticate = super.authenticate( authentication );
+    Authentication decodedAuth = new UsernamePasswordAuthenticationToken(
```

Comment:
Considering the unit test that fail, maybe we should check if the parameter is null!